Resolves the following byte-compiler warning:

lib/hyperspec.el:1318:1: Warning: Alias for
    `common-lisp-hyperspec-glossary-function' should be declared before its
    referent

--- a/lib/hyperspec.el
+++ b/lib/hyperspec.el
@@ -1313,12 +1313,12 @@
 
 ;;;; Glossary
 
-(defvar common-lisp-hyperspec-glossary-function 'common-lisp-glossary-6.0
-  "Function that creates a URL for a glossary term.")
-
 (define-obsolete-variable-alias 'common-lisp-glossary-fun
   'common-lisp-hyperspec-glossary-function "2015-12-29")
 
+(defvar common-lisp-hyperspec-glossary-function 'common-lisp-glossary-6.0
+  "Function that creates a URL for a glossary term.")
+
 (defvar common-lisp-hyperspec--glossary-terms (make-hash-table :test #'equal)
   "Collection of glossary terms and relative URLs.")
 
